elearning-backend/database/migrations/2022_07_07_092503_create_user_profiles_table.php

```diff
@@ -17,7 +17,7 @@ public function up()
             $table->id();
             $table->string('lastname');
             $table->string('firstname');
-            $table->string('middlename');
+            $table->string('middlename')->nullable();
```

Comment:
- [ ] create new migration file

Reply:
Reverted previous change and added new migration file created to update columns.